packages/admin/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from './admin'
```

Comment:
that's it, that's all there is... @andrewxhill lemme know if this * thing isn't nice for docs etc.

Reply:
well.. doesn't matter right? since we wouldn't update docs

Reply:
good point